[TEST-FAILURE] atomiclong.AtomicLongAdvancedTest.testMultipleThreadAtomicLong

```
java.lang.AssertionError: CountDownLatch failed to complete within 50 seconds , count left: 10
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:717)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:710)
	at com.hazelcast.concurrent.atomiclong.AtomicLongAdvancedTest.testMultipleThreadAtomicLong(AtomicLongAdvancedTest.java:45)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-IbmJDK1.6/com.hazelcast$hazelcast/656/testReport/junit/com.hazelcast.concurrent.atomiclong/AtomicLongAdvancedTest/testMultipleThreadAtomicLong/

--- a/hazelcast/src/test/java/com/hazelcast/concurrent/atomiclong/AtomicLongAdvancedTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/concurrent/atomiclong/AtomicLongAdvancedTest.java
@@ -42,7 +42,7 @@ public class AtomicLongAdvancedTest extends HazelcastTestSupport {
                 }
             }.start();
         }
-        assertOpenEventually(countDownLatch, 50);
+        assertOpenEventually(countDownLatch, 300);
         assertEquals(0, atomicLong.get());
     }
 

